metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -253,69 +253,295 @@
         </value-attributes>
     </property>
     <property>
-        <name>global-json</name>
-        <display-name>global.json template</display-name>
-        <description>This is the jinja template for global.json file</description>
-        <value>
-{
-"es.clustername": "{{ es_cluster_name }}",
-"es.ip": "{{ es_url }}",
-"es.date.format": "yyyy.MM.dd.HH",
-"parser.error.topic": "indexing"
-}
-        </value>
+        <name>es_date_format</name>
+        <description>Elasticsearch Date Format</description>
+        <value>yyyy.MM.dd.HH</value>
+        <display-name>Elasticsearch Date Format</display-name>
+    </property>
+    <property>
+        <name>parser_error_topic</name>
+        <description>Parser Error Topic</description>
+        <value>indexing</value>
+        <display-name>Parser Error Topic</display-name>
+    </property>
+    <property>
+        <name>enrichment_workers</name>
+        <description>Number of Enrichment Topology Workers</description>
+        <value>1</value>
+        <display-name>Enrichment Workers</display-name>
```

Comment:
It does not seem like changes I make in Ambari are being propagated into the running topologies.  

Here is what I am seeing. After deploying in Full Dev, I went into Ambari and changed some Enrichment properties like "Enrichment Workers".  The Enrichment component was then restarted.  

* In Ambari, the "Enrichment Workers" value is set to 2.  This is what I expect.

* When I look on disk `/usr/metron/0.4.0/config/enrichment.properties` the value is still the default of 1.  I would expect to see 2 here.
        ```
         enrichment.workers=1
        ```

* The Storm UI also indicates that the Enrichment topology is running with a `topology.workers` value of 1.  I would expect to see 2 here.